src/scales/scale.time.js

```diff
@@ -146,16 +146,26 @@ module.exports = function(Chart) {
 			minUnit: 'millisecond',
 
 			// defaults to unit's corresponding unitFormat below or override using pattern string from http://momentjs.com/docs/#/displaying/format/
+			// Use more detailed date strings on major ticks and summary info on minor ticks
 			displayFormats: {
-				millisecond: 'h:mm:ss.SSS a', // 11:20:01.123 AM,
-				second: 'h:mm:ss a', // 11:20:01 AM
-				minute: 'h:mm a', // 11:20 AM
+				millisecond: '.SSS', // .123,
+				second: 'h:mm:ss a', // 11:20:01
+				minute: 'h:mm', // 11:20
 				hour: 'hA', // 5PM
-				day: 'MMM D', // Sep 4
+				day: 'D', // 4
 				week: 'll', // Week 46, or maybe "[W]WW - YYYY" ?
-				month: 'MMM YYYY', // Sept 2015
-				quarter: '[Q]Q - YYYY', // Q3
-				year: 'YYYY' // 2015
+				month: 'MMM', // Sept
+				quarter: '[Q]Q', // Q3
+				year: 'YYYY', // 2015
+				major: {
```

Comment:
That's weird to have it under `displayFormats`, can we use the recently introduced [`major` option](/chartjs/Chart.js/blob/master/src/core/core.scale.js#L56) instead?

```javascript
ticks: {
    major: {
        displayFormat: { ... }
    }
}
```